app/src/main/java/lt/getpet/getpet/OnboardingActivity.kt

```diff
@@ -0,0 +1,50 @@
+package lt.getpet.getpet
+
+import android.os.Bundle
+import kotlinx.android.synthetic.main.activity_onboarding.*
+import lt.getpet.getpet.adapters.OnboardingAdapter
+import lt.getpet.getpet.navigation.NavigationManager
+import javax.inject.Inject
+import androidx.viewpager.widget.ViewPager
+
+
+
+class OnboardingActivity : BaseActivity() {
+
+    @Inject
+    lateinit var navigationManager: NavigationManager
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_onboarding)
+
+        val adapter = OnboardingAdapter(this, supportFragmentManager)
+
+        onboarding_viewpager.adapter = adapter
```

Comment:
Inline variable as it's not used anywhere else.

Reply:
Done.